### PR TITLE
components: 02-elements: cookie-consent: add dialog

### PR DESCRIPTION
**Pull request by BibianaC**
relates to #63

### CODE REVIEW

File: src/components/02-elements/cookie-consent/cookie-consent.scss [hunk 1]
```diff
@@ -1,7 +1,9 @@
 .cookie-consent {
```
Comment (michaelwood) [inline src/components/02-elements/cookie-consent/cookie-consent.scss:1]: Did you mean to commit these changes in this PR? I thought it was done in #62

Comment (BibianaC) [reply src/components/02-elements/cookie-consent/cookie-consent.scss:1]: This branch was checkout from the #62 but was done before the `index` was added. That is why there is a conflict here. Let me fix this first.

Comment (BibianaC) [reply src/components/02-elements/cookie-consent/cookie-consent.scss]: @michaelwood I have fixed the conflict so the file is no longer in this PR.

**Comment by BibianaC** [comment]
Done in https://github.com/ThreeSixtyGiving/360-ds/pull/65

**Pull request closed without merging**